BudgetBudget/View/MonthSelectorView.swift

```diff
@@ -32,6 +32,10 @@ struct MonthSelectorView: View {
                                     Text(month)
                                         .foregroundStyle(.primary)
                                         .padding(5)
+                                        // FIXME: why doesn't this work?
+//                                        .backgroundStyle(scrollTarget == id ? .selection : .background)
+//                                        .background(scrollTarget == id ? .selection : .background)
+                                        .fontWeight(scrollTarget == id ? .bold : .regular)
                                         .overlay(alignment: .bottom) {
                                             if (year == MonthSelectorView.currentYear && month == MonthSelectorView.currentMonth) {
```

Comment:
I prefer the UX of the original BudgetBudget where the currently displayed months are highlighted with this bar below. I already implemented the bar UI already as the overlay. Instead of making it visible for the currentMonth (which it was for testing) it should be bound to the (three) visible months.

I would then rather highlight the current month in bold (if at all) since this is less important than the currently shown one.

Reply:
Didn't even notice the blue bar. Maybe it's only for me, but it's so thin that it's barely noticeable even now I'm aware of it. I'll extend it to all displayed months and make it just a tiny bit thicker.